etc/proto/Dockerfile

```diff
@@ -12,8 +12,6 @@ RUN \
 RUN unzip protoc.zip -d /
 RUN cp -r /include /bin
 
-RUN go get -u -v github.com/golang/protobuf/proto
```

Comment:
this doesn't seem to be doing anything since we're using gogoproto in our protoc process